functions/src/tasks/index.js

```diff
@@ -35,3 +35,11 @@ exports.createTask = (req, res) => {
     .then(() => this.getTasks(req, res))
     .catch(err => res.status(500).send('Error creating task: ' + err.message))
 }
+
+exports.getSingleTask = (req, res) => {
+  connectToFirestore()
+  const singleTask = req.params
+  db.collection('tasks').get(singleTask)
```

Comment:
Check firestore docs for how to get a single document

Reply:
```exports.getSingleTask = (req, res) => {
  connectToFirestore()
  const { taskId } = req.params
  db.collection('tasks')
    .doc(taskId)
    .get()
    .then(singleTask => res.send(singleTask.data()))
}```